mycroft/configuration/mycroft.conf

```diff
@@ -237,7 +237,9 @@
   },
 
   // Level of logs to store, one of  "CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"
-  "log_level": "DEBUG",
+  // Must be set in the system or user config, this or remote will not affekt
```

Comment:
Two things:
affekt -> affect

If this is not specified in any .conf file, what will the level be?  Looking at the code below it appears the default is INFO, but that isn't totally clear.  How about this text:

// Level of logs to store, one of  "CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"
// NOTE:  This configuration setting is special and can only be changed in the
// SYSTEM or USER configuration file, it will not be read if defined in the
// DEFAULT (here) or in the REMOTE mycroft.conf settings.
// If not defined, the default log level is INFO.